packages/shared/src/modules/notifications/libs/enums/entity-type.enum.ts

```diff
@@ -0,0 +1,5 @@
+const EntityType = {
```

Comment:
```suggestion
const NotificationEntityType = {
```
don't forget about file name

Reply:
So, generally i just need to change type from EntityType to NotificationEntityType